.github/workflows/ci.yml

```diff
@@ -24,20 +24,114 @@ on:
 
 jobs:
   check-semantic-version:
-    if: github.event.pull_request.draft == false
+    if: github.ref != 'refs/heads/main'
     uses: octue/workflows/.github/workflows/check-semantic-version.yml@main
     with:
       path: pyproject.toml
       breaking_change_indicated_by: minor
 
-  run-tests:
+  check-main-merge:
```

Comment:
GitHub already does this check for you and shows you at the bottom of the pull request

Reply:
That's true. I'd thought that this might be a useful gate to stop the more expensive tests from running if there's no point (ie no point testing if the branch isn't up to date)